server/hpccloud/server/file.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from girder.api import access
+from girder.api.describe import Description, describeRoute
+from girder.utility.progress import ProgressContext
+from girder.api.rest import getBodyJson, getCurrentUser, ModelImporter
+
+
+def move_files_to_assetstore(files, totalsize):
+    user, token = getCurrentUser(True)
+    assetstore = ModelImporter.model('assetstore').getCurrent()
+    upload = ModelImporter.model('upload')
+    with ProgressContext(True, interval=1.0, message=files[0]['name'],
+                         user=user, token=token,
+                         total=totalsize, title='Moving files') as ctx:
+        for file in files:
+            ctx.update(message=file['name'])
```

Comment:
You're right we're not displaying it anywhere. I'm not sure this line has the intended effect either.